pkg/models/dashboards.go

```diff
@@ -323,15 +323,13 @@ type GetDashboardSlugByIdQuery struct {
 	Result string
 }
 
-type IsDashboardProvisionedQuery struct {
+type GetProvisionedDashboardDataByIdQuery struct {
```

Comment:
I changed this so instead of bool it return full data or nil if the row does not exist so it can be still used to check for existence but also returns data.